base/applications/regedit/main.c

```diff
@@ -191,7 +191,11 @@ BOOL TranslateChildTabMessage(PMSG msg)
 
     if (msg->wParam != VK_TAB) return FALSE;
     if (GetParent(msg->hwnd) != g_pChildWnd->hWnd) return FALSE;
+#ifdef __REACTOS__
+    PostMessageW(hFrameWnd, WM_COMMAND, ID_SWITCH_PANELS, 0);
+#else
     PostMessageW(g_pChildWnd->hWnd, WM_COMMAND, ID_SWITCH_PANELS, 0);
+#endif
```

Comment:
Since you guarded the thing, what is the value of `g_pChildWnd->hWnd`, which window does it correspond to, and how does it compare with `hFrameWnd` ?

Reply:
https://github.com/reactos/reactos/blob/7cc89e520c3ac4182e9f6db13610e01a5e54efe5/base/applications/regedit/childwnd.c#L363
https://github.com/reactos/reactos/blob/7cc89e520c3ac4182e9f6db13610e01a5e54efe5/base/applications/regedit/framewnd.c#L1356
https://github.com/reactos/reactos/blob/7cc89e520c3ac4182e9f6db13610e01a5e54efe5/base/applications/regedit/framewnd.c#L1023

Reply:
![image](https://user-images.githubusercontent.com/2107452/202575488-ac4a202b-9d12-437b-976b-87a68e7f1ea7.png)

Class `REGEDIT_FRAME` is the frame window.
Class `REGEDIT` is the child window.
The treeview is on the child window.